sidecars/rust/etc/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Is this file executable (i.e. `chmod +x` was run)?

Reply:
yes it is executable
![Selection_080](https://user-images.githubusercontent.com/650571/101140069-b8a40400-3612-11eb-8c2c-d60344553daf.png)